ci-operator/step-registry/openshift-qe/installer/featuregate/openshift-qe-installer-featuregate-chain.metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+	"path": "openshift-qe/installer/featuregate/openshift-qe-installer-featuregate-chain.yaml",
+	"owners": {
+		"approvers": [
+			"afcollins",
```

Comment:
can we get lgtm from one of the QE perf/scale approvers as well for posterity given we are adding their team here as OWNERS for this along with central perf/scale team..